py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -358,7 +358,7 @@ def aten_ops_grid(
     )
 
 
-@dynamo_tensorrt_converter(torch.ops.aten.relu.default)
+@dynamo_tensorrt_converter(torch.ops.aten.relu.default, dynamic=True)
```

Comment:
Are there cases where we want `dynamic=None` to mean _both_ dynamic and not dynamic, and reserve `dynamic=True` to imply _only_ dynamic?

Reply:
I think it should be a support vs not support since any dynamic shape converter should be able to support both right?

Reply:
Current assumption is `dynamic=True` implies it supports both static and dynamic. I'm not aware of any converters which only work for dynamic (but not static).